assets/src/dashboard/app/views/shared/bodyViewOptions.js

```diff
@@ -24,7 +24,11 @@ import PropTypes from 'prop-types';
  * Internal dependencies
  */
 import { Dropdown, ListBar } from '../../../components';
```

Comment:
Super nit, can you update the grid/list cursor to be a pointer?
![grid_icon](https://user-images.githubusercontent.com/40646372/79799957-36c7ca80-8310-11ea-84f7-ab0c62297585.gif)